src/MICore/MIResults.cs

```diff
@@ -730,12 +730,20 @@ private Results ParseResultList(Span listStr, ResultClass resultClass = ResultCl
                     {
                         return i == s.Extent;
                     }, listStr, out rest);
+
+            Results results = new Results(resultClass, list);
+
             if (!rest.IsEmpty)
             {
                 ParseError("trailing chars", rest);
-                return null;
+
+                if (rest.Length > 1000)
+                {
+                    rest = new Span(rest.Start, 1000);    // don't show more than 1000 chars
```

Comment:
is there a reason for 1000 characters? should it be a const value you use in both places? is that too much regardless?

Reply:
Was doing the same limit in ParseError https://github.com/microsoft/MIEngine/blob/95f12798e92a046b8a0a47b087319b6c569f4f7e/src/MICore/MIResults.cs#L1206

Reply:
can we make that a static so you we can change that if need to?